src/io/dsv.rs

```diff
@@ -189,15 +189,15 @@ Boston;United States;4628910
                 .and_then(|u64| usize::try_from(u64).ok())
                 .and_then(|usize| dict.entry(usize))
                 .unwrap(),
-            "Boston"
+            "<Boston>"
```

Comment:
It might be a good test-case to leave one of the strings without the "< >" angles (which seems to work fine as well)